nuget/lib/dependabot/nuget/update_checker/version_finder.rb

```diff
@@ -234,7 +234,7 @@ def fetch_paginated_v2_nuget_listings(url_details, results = {})
             url_details = url_details.dup
             # Some Nuget repositories, such as JFrog's Artifactory, URL encode the "next" href
             # link in the paged results. If the href is not URL decoded, the paging parameters
-            # are ignored and the first page is always returned. 
```

Comment:
Oof! That's what I get for editing directly in the Web UI instead of my editor. Thanks @Nishnha !